packages/eslint-config/index.js

```diff
@@ -179,16 +179,46 @@ module.exports = {
     'prefer-object-spread': 'error',
     'space-in-parens': ['error', 'never'],
     'spaced-comment': 'error',
-    'valid-jsdoc': [
-      'error',
+    'jsdoc/check-alignment': 'error',
+    'jsdoc/check-indentation': 'error',
+    'jsdoc/check-param-names': 'warn',
+    'jsdoc/check-syntax': 'error',
+    'jsdoc/multiline-blocks': 'error',
+    'jsdoc/newline-after-description': ['error', 'never'],
+    'jsdoc/no-bad-blocks': 'error',
+    'jsdoc/no-defaults': 'error',
+    'jsdoc/no-types': 'error',
+    'jsdoc/require-asterisk-prefix': 'error',
+    'jsdoc/require-description-complete-sentence': [
```

Comment:
Would this rule work if we just disable it for the `index.ts` files where we describe packages?

Reply:
Yes, it works for everything thats publicly exported